src/components/Login.js

```diff
@@ -1,8 +1,19 @@
 import React, { Component } from "react";
-import { Link } from "react-router-dom";
+import { Link } 
+from "react-router-dom";
+import { Button, Form, FormGroup, Label } from "reactstrap";
 
-import { Button, Form, FormGroup, Label, Input } from "reactstrap";
+import { Formik, Field, ErrorMessage } from "formik";
+import * as Yup from "yup";
 
+const LoginSchema = Yup.object().shape({
+  username: Yup.string()
+    .min(6, "Must be 6 characters or less")
```

Comment:
Isn't it minimum six characters?

Reply:
not mandatory 😀

Reply:
`Must be 6 chars or less` means it can be a maximum of six characters, while [`.string().min(6)`](https://github.com/jquense/yup#stringminlimit-number--ref-message-string--function-schema) means a minimum of six characters.